content/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file.md

```diff
@@ -662,53 +662,99 @@ updates:
+| `widen`| Widen the allowed version requirements to include both the new and old versions, when possible. Typically this only increases the maximum allowed version requirement. |
+
+Example:
+
+<table>
+    <thead>
+        <tr>
+            <th align="center">Constraint</th>
+            <th align="center">Current Version</th>
+            <th align="center">New Version</th>
+            <th>Strategy</th>
+            <th>New Constraint</th>
+        </tr>
+    </thead>
+    <tbody>
+        <tr>
+            <td rowspan="3" align="center"><code>^1.0.0</code></td>
+            <td rowspan="3" align="center"><code>1.0.0</code></td>
+            <td rowspan="3" align="center"><code>1.2.0</code></td>
+            <td align="center"><code>'widen'</code></td>
+            <td><code>^1.0.0</code></td>
+        </tr>
+        <tr>
+            <td align="center"><code>'increase'</code></td>
+            <td><code>^1.2.0</code></td>
+        </tr>
+        <tr>
+            <td align="center"><code>'increase-if-necessary'</code></td>
+            <td><code>^1.0.0</code></td>
+        </tr>
+      <tr>
+            <td rowspan="3" align="center"><code>^1.0.0</code></td>
+            <td rowspan="3" align="center"><code>1.0.0</code></td>
+            <td rowspan="3" align="center"><code>2.0.0</code></td>
+            <td align="center"><code>'widen'</code></td>
+            <td><code>&gt;=1.0.0 &lt;3.0.0</code></td>
+        </tr>
+        <tr>
+            <td align="center"><code>'increase'</code></td>
+            <td><code>^2.0.0</code></td>
+        </tr>
+        <tr>
+            <td align="center"><code>'increase-if-necessary'</code></td>
+            <td><code>^2.0.0</code></td>
+        </tr>
+    </tbody>
+</table>
 
 Use the `versioning-strategy` option to change this behavior for supported package managers.
 
 {% data reusables.dependabot.option-affects-security-updates %}
 
 Available update strategies
 
-| Option | Supported by | Action |
-|--------|--------------|--------|
-| `lockfile-only` | `bundler`, `cargo`, `composer`, `mix`, `npm`, `pip` | Only create pull requests to update lockfiles. Ignore any new versions that would require package manifest changes. |
-| `auto` | `bundler`, `cargo`, `composer`, `mix`, `npm`, `pip` | Follow the default strategy described above.|
-| `widen`| `composer`, `npm` | Relax the version requirement to include both the new and old version, when possible. |
-| `increase`| `bundler`, `composer`, `npm` | Always increase the version requirement to match the new version. |
-| `increase-if-necessary` | `bundler`, `composer`, `npm` | Increase the version requirement only when required by the new version. |
+| Ecosystem | Supported _versioning-strategies_ | Default strategy | Additional notes |
+|-----------|-----------------------------------|------------------|------------------|
+| `bundler` | `auto`, `increase`, `increase-if-necessary`, `lockfile-only` | `auto` |
```

Comment:
`increase` and `increase-if-necessary` are misnamed. I can't find any record of them in our codebase.

I think these were meant to be `bump_versions` and `bump_versions_if_necessary`, which we support in [our dry-runner](https://github.com/dependabot/dependabot-core/blob/e11608fd17da3275cefca5e22eb01d01eb62fe09/bin/dry-run.rb#L196-L197) and were added to the bundler ecosystem in [Dependabot v0.85.2](https://github.com/dependabot/dependabot-core/blob/e11608fd17da3275cefca5e22eb01d01eb62fe09/CHANGELOG_ARCHIVE_2017_TO_2018.md?plain=1#L96-L99)

Reply:
🤦 

I'm surprised that the incorrect config values weren't blowing up...probably they were silently getting swallowed as no-ops. I'll update this.

Reply:
Actually turns out the `increase-if-necessary` and `increase` terminology is correct. The `bump_versions` and `bump_versions_if_necessary` are just internal naming, but the proper values for the configuration are the ones documented 👍.